src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -115,7 +115,7 @@ public static function getStyleDefinition($name)
             self::$styles = self::initStyles();
         }
 
-        if (!self::$styles[$name]) {
+        if (empty(self::$styles[$name])) {
```

Comment:
Reading the code, it looks like it is defined with an object or not defined at all. So, I think we should keep `isset()` as before in the new `resolveStyle()` method and change this one to use `isset()` as well. Whenever possible, I tend to avoid using `empty()`.